app/src/Services/modules/projects/index.ts

```diff
@@ -1,24 +1,31 @@
+import { ImageURISource } from 'react-native'
 import { api } from '../../api'
 import fetchAll from './fetchAll'
 
 export const projectsApi = api.injectEndpoints({
   endpoints: build => ({
     fetchAll: fetchAll(build),
   }),
-  overrideExisting: false,
+  overrideExisting: true,
 })
 
 export const { useLazyFetchAllQuery } = projectsApi
 
 export interface Project {
-  name: string
-  client: string
-  role: string
-  description: string
-  skills: string[]
-  hours: number
-  required: boolean
-  buddying: boolean
+  ResData: {
+    res_id: string
+    projectType: string
+    jobRole: string
+    description: string
+    candidateCoreSkills: string
+    candidateTime: string
+    suitableForBuddy: boolean
+  },
+  ItData: {
+    projectName: string
+    charityName: string
+    charityVideo: ImageURISource
+  }
```

Comment:
I think this is probably how the data for projects is coming through from the API, but wondering if we can get the backend folks to flatten these 2 objects into a single object rather than having these 2 nested `ResData` and `ItData` objects so it would be more like this?
```suggestion
res_id: string
projectType: string
jobRole: string
description: string
candidateCoreSkills: string
candidateTime: string
suitableForBuddy: boolean
projectName: string
charityName: string
charityVideo: ImageURISource
```

That way, when we access the project data (in `app/src/Components/Project/ProjectSummary.tsx`) we don't need to dig 2 levels deep into the object:
`<ProjectHeading title={project.projectName} />`
instead of:
`<ProjectHeading title={project.ItData.projectName} />`

We could maybe add a ticket in the backlog to look at doing the above separately?

Reply:
Good point, it would be neater that way. As discussed in the team meeting this morning a new ticket will be raised for this.